### DISCUSSION TITLE
悪魔エゴのような「元素+付加価値+軽ペナルティ」のエゴを他元素にも拡張する案

### DISCUSSION THREAD

**Discussion by RL-Nadir** [hengband/hengband#325] [Ideas]
投稿内容の性質を鑑み、Issueから転載しました。エゴ名称の苦しさの指摘を既にdiscordにていただいています。

-------------

武器、防具の悪魔エゴを、比較的軽微なペナルティと引き換えに能力・耐性の付加価値をつけた元素装備と位置づけ、
火炎+地獄に限られている現状から、他の三属性にも拡張することが狙いです。
スペックありきで考えたため、これにふさわしく当てられるネタのレパートリーがなく、苦しいです。
もし採用の際は、遠慮なくより適切なものに変更ください。

○武器エゴ

N:xxx:(死霊)
E:(Wraith)
X:24:25
W:0:30:0:15000
C:6:6:-20:4
F:WIS | BRAND_COLD | CURSED | POWERFUL |
F:RES_COLD | RES_DARK | SEE_INVIS | ESP_UNDEAD | XTRA_DICE

N:xxx:(鳴神)
E:(Narukami)
X:24:25
W:0:30:0:15000
C:6:6:-20:4
F:INT | BRAND_ELEC | CURSED | POWERFUL |
F:RES_ELEC | RES_SOUND | SEE_INVIS | ESP_DRAGON | XTRA_DICE

N:xxx:(蛟)
E:(Mizuchi)
X:24:25
W:0:30:0:15000
C:6:6:-20:4
F:WIZ | BRAND_ACID | CURSED | POWERFUL |
F:RES_ACID | RES_POISON | SEE_INVIS | ESP_ANIMAL | XTRA_DICE

e_info外で処理されている追加処置は(悪魔)エゴと同じに。

○防具エゴ(胴体枠）

N:xxx:死霊の
E:of Wraith
X:31:30
W:0:40:0:15000
C:0:0:11:3
F:STR | WIS | CON |
F:RES_COLD | RES_DARK | ESP_UNDEAD | SH_COLD | CURSED | RANDOM_CURSE2 | POWERFUL |
F:IGNORE_FIRE | XTRA_RES

N:xxx:鳴神の
E:of Narukami
X:31:30
W:0:40:0:15000
C:0:0:11:3
F:DEX | INT | CON |
F:RES_ELEC | RES_SOUND | ESP_DRAGON | SH_ELEC | CURSED | RANDOM_CURSE2 | POWERFUL |
F:IGNORE_FIRE | XTRA_RES

N:xxx:蛟の
E:of Mizuchi
X:31:30
W:0:40:0:15000
C:0:0:11:3
F:DEX | WIZ | CON |
F:RES_ACID | RES_POISON | ESP_ANIMAL | SH_COLD | CURSED | RANDOM_CURSE2 | POWERFUL |
F:IGNORE_FIRE | XTRA_RES

e_info外で処理されている追加処置は「悪魔の」エゴと同じに。

-----------
3/1追記　頭枠にもあったのを忘れていました

○防具エゴ(頭枠）
N:XX:死霊の
E:of Wraith
X:33:30
C:0:0:10:3
W:0:30:0:12000
F:WIS | SEE_INVIS |
F:RES_COLD | RES_DARK | ESP_UNDEAD | SH_COLD | CURSED | RANDOM_CURSE2 | POWERFUL | 
F:IGNORE_FIRE | XTRA_RES

N:XX:鳴神の
E:of Narukami
X:33:30
C:0:0:10:3
W:0:30:0:12000
F:INT | SEE_INVIS |
F:RES_ELEC | RES_SOUND | ESP_DRAGON | SH_ELEC | CURSED | RANDOM_CURSE2 | POWERFUL | 
F:IGNORE_FIRE | XTRA_RES

N:XX:蛟の
E:of Mizuchi
X:33:30
C:0:0:10:3
W:0:30:0:12000
F:WIS | SEE_INVIS |
F:RES_ACID | RES_POISON | ESP_ANIMAL | SH_COLD | CURSED | RANDOM_CURSE2 | POWERFUL | 
F:IGNORE_FIRE | XTRA_RES

e_info外で処理されている追加処置は「悪魔の」エゴと同じに。

**Comment by ghost** [comment]
デメリット付きエゴを増やす場合は何らかの配慮(手加減)が欲しいです。
現状でも悪魔武器／防具、モルグル武器／防具、妖刀武器などが存在し、6/12FRQ、ヘビの舌などの高級品ドロップユニークから稀によく出てきます。(巷ではそういう場合Q@yという風潮かもしれませんが)
RFQや高級品ドロップの敵から「騒音の」、「のろまの」といったマイナス効果エゴが出たことは(今までプレーした限りでは)ないので、もしそれらのドロップに何らかの配慮がされているのでれば、新規のデメリット付きエゴにも同様の手心を加えることをご検討いただければ幸いです。

**Reply by RL-Nadir** [reply]
提案者です。コメントありがとうございます。
他のエゴアイテムが圧迫されることについては全く考えが及んでいませんでした。懸念されていることはもっともなことだと思います。
なので、このようなデメリット付きエゴアイテムが追加される場合は、個々のレア度を調整することで、シリーズとして従来並の出やすさになる（＝既存の良質エゴに割りを食わせない）ような調整をお願いしたいと思います。

**Reply by iks3** [reply]
#483 にて、呪いの付くエゴは上質ドロップには出てこなくなりました。

**Comment by deskull-m** [comment]
ひとまず、もらった案の中におかしいフラグ表記や属性の違う内容が多かったのに加え、鎧と頭防具は個別に実装する必要があったので以下のような調子で調整してみました。
7expさんの憂慮される他の確率への影響ですが、ヴィザードモードのoコマンドのexcellent生成で見る限り大きな枠の逼迫を起こす様子はなかったように思えます。とはいえ、その辺の感覚はひとそれぞれとは思いますので、一度Alpha10でこのまま実装し、プルリクエストとIssueは閉じつつ、ここで継続して議論できると幸いです。

> N:241:(死霊)
> E:(Wraith)
> X:24:25
> W:0:30:0:15000
> C:6:6:-20:4
> F:WIS | BRAND_COLD | CURSED | POWERFUL |
> F:RES_COLD | RES_DARK | SEE_INVIS | ESP_UNDEAD | XTRA_DICE
> 
> N:242:(鳴神)
> E:(Narukami)
> X:24:25
> W:0:30:0:15000
> C:6:6:-20:4
> F:INT | BRAND_ELEC | CURSED | POWERFUL |
> F:RES_ELEC | RES_SOUND | SEE_INVIS | ESP_DRAGON | XTRA_DICE
> 
> N:243:(蛟)
> E:(Mizuchi)
> X:24:25
> W:0:30:0:15000
> C:6:6:-20:4
> F:WIS | BRAND_ACID | CURSED | POWERFUL |
> F:RES_ACID | RES_POIS | SEE_INVIS | ESP_ANIMAL | XTRA_DICE
> 
> N:244:死霊の
> E:of Wraith
> X:31:30
> W:0:40:0:15000
> C:0:0:11:3
> F:STR | WIS | CON |
> F:RES_COLD | RES_DARK | ESP_UNDEAD | SH_COLD | CURSED | RANDOM_CURSE2 | POWERFUL |
> F:IGNORE_COLD | XTRA_RES
> 
> N:245:鳴神の
> E:of Narukami
> X:31:30
> W:0:40:0:15000
> C:0:0:11:3
> F:DEX | INT | CON |
> F:RES_ELEC | RES_SOUND | ESP_DRAGON | SH_ELEC | CURSED | RANDOM_CURSE2 | POWERFUL |
> F:IGNORE_ELEC | XTRA_RES
> 
> N:246:蛟の
> E:of Mizuchi
> X:31:30
> W:0:40:0:15000
> C:0:0:11:3
> F:DEX | WIS | CON |
> F:RES_ACID | RES_POIS | ESP_DRAGON | CURSED | RANDOM_CURSE2 | POWERFUL |
> F:IGNORE_ACID | XTRA_RES
> 
> N:247:死霊の
> E:of Wraith
> X:33:30
> W:0:40:0:15000
> C:0:0:11:3
> F:STR | WIS | CON |
> F:RES_COLD | RES_DARK | ESP_UNDEAD | SH_COLD | CURSED | RANDOM_CURSE2 | POWERFUL |
> F:IGNORE_COLD | XTRA_RES
> 
> N:248:鳴神の
> E:of Narukami
> X:33:30
> W:0:40:0:15000
> C:0:0:11:3
> F:DEX | INT | CON |
> F:RES_ELEC | RES_SOUND | ESP_DRAGON | SH_ELEC | CURSED | RANDOM_CURSE2 | POWERFUL |
> F:IGNORE_ELEC | XTRA_RES
> 
> N:249:蛟の
> E:of Mizuchi
> X:33:30
> W:0:40:0:15000
> C:0:0:11:3
> F:DEX | WIS | CON |
> F:RES_ACID | RES_POIS | ESP_DRAGON | CURSED | RANDOM_CURSE2 | POWERFUL |
> F:IGNORE_ACID | XTRA_RES
> 
>

**Reply by deskull-m** [reply]
他件との並行や要調整の意見がDiscordに上がったのでAlpha10での採用は取りやめます。

**Reply by iks3** [reply]
(死霊)は、現在のモルグルを強化で対応でよいかなと思います。
蛟は、(貪食)/(Omnivorous)あたりの名称で、必ず速消化(呪いで付くアレ)と急回復が付く感じにできればそれっぽそうな気がします。
鳴神は英語名をどうにかしたいですねえ。
ちょっと調整考えてみます。

**Reply by ghost** [reply]
> 鳴神は英語名をどうにかしたいですねえ

雷神＆呪いというニュアンスならば「ハオカー」(Heyoka/Haoka)とかどうでしょ？
コンシューマーゲームなんかでも使われてる単語なので海外でも知名度はあるかも。
https://ja.wikipedia.org/wiki/%E3%83%8F%E3%82%AA%E3%82%AB%E3%83%BC

**Reply by RL-Nadir** [reply]
呪いエゴ生成の調整ありがとうございます。
このアイディアで良質エゴが割りを食うことがなくなり、安堵しています。

死霊の調整案は名称の意味合いも重なっていますので良いと思います。
貪食は原案よりそれらしいし、呪いもいい感じですね。

ハオカー・・・集合知に助けられました。悪い雷さまネタありがとうございます。
英訳と和訳が1:1でなくていいなら、鳴神よりも、雷獣とか、怨霊としての天神でもいいかもしれないです。
このあたりはもうお好みでお願いします。
